tests/Provider/SessionProviderTest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Web\Tests\Provider;
+
+use Yiisoft\Yii\Web\Session\SessionInterface;
+use Yiisoft\Yii\Web\Tests\TestCase;
+
+final class SessionProviderTest extends TestCase
+{
+    public function testProviderConfig(): void
+    {
+        $this->assertNotEmpty($this->container->get(SessionInterface::class));
```

Comment:
It's better to use `assertInstanceOf` for these.

Reply:
done.